src/drivers/src/stm32f4xx.c

```diff
@@ -35,6 +39,11 @@ HAL_StatusTypeDef CANlib_TransmitFrame(Frame *frame, CANlib_Bus_T bus) {
   pHeader.IDE = frame->extended ? CAN_ID_EXT: CAN_ID_STD;
   pHeader.RTR = CAN_RTR_DATA; // Data frame (as opposed to a remote frame)
   pHeader.TransmitGlobalTime = DISABLE; // Don't replace last 2 bytes of data with TX time.
+#ifdef USING_LOGGING_CALLBACK
```

Comment:
@equationcrunchor this seems like something that should not have been approved and that should be reversed...